From 389a3c294ae914efd1681f5355b07ed8b439678d Mon Sep 17 00:00:00 2001
From: Randy Dunlap <rdunlap@infradead.org>
Date: Wed, 29 Jul 2026 22:17:10 -0700
Subject: [PATCH 12529/15333] crypto: ccp - don't abuse kernel-doc comment
 format

Use plain C "/*" notation for comments that are not in kernel-doc format
to avoid kernel-doc warnings:

Warning: include/uapi/linux/psp-sfs.h:18 expecting prototype for SFS().
 Prototype was for PAYLOAD_NAME_SIZE() instead
Warning: include/uapi/linux/psp-sfs.h:46 This comment starts with '/**',
 but isn't a kernel-doc comment.
 * Seamless Firmware Support (SFS) IOC

Fixes: 648dbccc03a0 ("crypto: ccp - Add AMD Seamless Firmware Servicing (SFS) driver")
Signed-off-by: Randy Dunlap <rdunlap@infradead.org>
Acked-by: Tom Lendacky <thomas.lendacky@amd.com>
Signed-off-by: Herbert Xu <herbert@gondor.apana.org.au>
---
 include/uapi/linux/psp-sfs.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/uapi/linux/psp-sfs.h b/include/uapi/linux/psp-sfs.h
index 94e51670383c..fe9402c8a575 100644
--- a/include/uapi/linux/psp-sfs.h
+++ b/include/uapi/linux/psp-sfs.h
@@ -12,7 +12,7 @@
 
 #include <linux/types.h>
 
-/**
+/*
  * SFS: AMD Seamless Firmware Support (SFS) interface
  */
 
@@ -43,7 +43,7 @@ struct sfs_user_update_package {
 	__u32	sfs_extended_status;
 } __packed;
 
-/**
+/*
  * Seamless Firmware Support (SFS) IOC
  *
  * possible return codes for all SFS IOCTLs:
-- 
2.55.0

